templates/learn.md

```diff
@@ -4,30 +4,39 @@ There are many ways to start learning Lean, depending on your background and
 taste. They are all fun and rewarding, but also difficult and
 occasionally frustrating. Proof assistants are still difficult to use,
 and you cannot expect to become proficient after one afternoon of
-learning. Note that all resources listed on that page are about Lean 4.
-There is no point learning Lean 3 at this stage.
+learning.
+
+All the resources listed on this page are about Lean 4.
+Some have Lean 3 versions, but there is no point learning Lean 3 at this stage.
 
 ## Hands-on approaches
 
 * Whatever your background, if you want to dive right away, you can play the
-  [Natural Number Game](https://adam.math.hhu.de/#/g/hhu-adam/NNG4).
-  This is an online interactive tutorial to Lean
+  [Natural Number Game](https://adam.math.hhu.de/#/g/hhu-adam/NNG4). 
+  This is an online interactive Lean tutorial
   focused on proving properties of the elementary operations on natural numbers.
+  Similar games for [Logic and Set Theory](https://adam.math.hhu.de/#/) are available.
 
-* For a faster paced and broader dive, you can get the
+* For a faster-paced and broader dive, you can get the
   [Glimpse of Lean tutorial](https://github.com/PatrickMassot/GlimpseOfLean).
   This contains four basic files covering some fundamental aspects of proving
   using Lean, and then independent topic files about elementary analysis,
   abstract topology and mathematical logic.
 
 ## Books
 
-* If you prefer reading a book (with exercises), the standard mathematics oriented reference is
+If you prefer reading a book (with exercises), there are a number of freely available Lean books.
+These are available as PDFs, but usually meant to be read interactively in VSCode,
+doing Lean exercises on the fly:
+
+* The [Hitchhiker's Guide to Logical Verification (2023 Edition)](https://github.com/blanchette/logical_verification_2023) is a good introduction to both the verification and programming aspects of Lean,
+with an associated VSCode project and exercises. [PDF here](https://github.com/blanchette/logical_verification_2023/blob/main/hitchhikers_guide.pdf).
```

Comment:
This should definitely not be the first suggestion. Also, I already added it to the list below.

Reply:
Fixed